src/layouts/Header/components/Navigator/Navigator.tsx

```diff
@@ -38,6 +42,15 @@ function Navigator({ currentUser }: NavigatorProps) {
     )
   }
 
+  if (ownerData && !ownerData.isCurrentUserPartOfOrg) {
+    return (
+      <div className="flex items-center">
+        <Avatar user={ownerData} />
+        <h2 className="mx-2 text-xl font-semibold">{ownerData?.username}</h2>
```

Comment:
don't need the optional chain here since we're checking ownerData existing in the condition